src/server/collisions.test.js

```diff
@@ -0,0 +1,42 @@
+const applyCollisions = require('./collisions');
+const Constants = require('../shared/constants');
+const Player = require('./player');
+const Bullet = require('./bullet');
+
+describe('applyCollisions', () => {
+  it('should not collide when outside radius', () => {
+    const distanceFromPlayer = Constants.BULLET_RADIUS + Constants.PLAYER_RADIUS + 1;
+    const players = [
+      new Player('1', 'guest1', 1000, 40),
+      new Player('2', 'guest2', 2000, 2000),
+    ];
+    const bullets = [
+      new Bullet('3', 1000 - distanceFromPlayer, 40, 0),
+      new Bullet('3', 1000 + distanceFromPlayer, 40, 0),
```

Comment:
make these two bullets have `parentID` of `'2'` since there isn't actually a player obj with ID 3?